### ISSUE TITLE
Problems with mesh

### ISSUE THREAD

**Issue by NoAH10833** [DerPopo/UABE#664]
Is there any way to edit a mesh in the same way we edit textures2d, I can't import ob files to use in the game

**Comment by whishad** [comment]
No, changing a mesh is a more complicated process than changing textures, to my surprise, there are a lot of videos on YouTube on uabe and about changing a mesh through it, you can just write in the YouTube search something similar to "how to change a mesh through uabe"
